ci/test-conformance-gke.sh

```diff
@@ -31,7 +31,8 @@ RUN_ALL=true
 RUN_SETUP_ONLY=false
 RUN_CLEANUP_ONLY=false
 TEST_SCRIPT_RC=0
-KUBE_CONFORMANCE_IMAGE_VERSION=auto
+# There is a problem with the netpol suite added in v1.21.0 when running on GKE. See #3762 for details.
+KUBE_CONFORMANCE_IMAGE_VERSION=v1.20.15
```

Comment:
I wonder if it's an Antrea issue, a GKE issue or a test issue

Reply:
Not sure yet, but here is what I've found so far:
1. The direct reason is that the test couldn't create a test Pod successfully
2. The Pod creation failure was because kube-apiserver couldn't find the associated serviceAccount (default)
```
unable to update pod netpol-4938-x/a: pods \"a\" is forbidden: error looking up service account netpol-4938-x/default: serviceaccount \"default\" not found
```

3. The default serviceAccount was created by kube-controller-manager, but sometimes took more than 4 seconds due to priorityandfairness 
```
I0511 03:45:36.681286      11 httplog.go:109] "HTTP" verb="POST" URI="/api/v1/namespaces/netpol-5472/serviceaccounts" latency="4.965865514s" userAgent="kube-controller-manager/v1.22.8 (linux/amd64) kubernetes/87bd527/system:serviceaccount:kube-system:service-account-controller" audit-ID="852ec2d3-805c-4097-ac75-b3a1ae409c33" srcIP="10.138.1.118:34048" resp=201 addedInfo="\npriorityandfairness=4.95694272s"
```

I suspect it was because GKE configures priorityandfairness differently, causing problems to tests that create multiple Namespaces and Pods in a short time.